.github/workflows/release.yml

```diff
@@ -22,18 +23,32 @@ jobs:
       - name: Install NPM packages
         working-directory: output-data
         run: npm ci
-      - name: Build
+      - name: Setup Java and Maven
+        uses: actions/setup-java@v1
+        with:
+          java-version: 17
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+      - name: Install gpg secret key
+        run: |
+          cat <(echo -e "${{ secrets.OSSRH_GPG_SECRET_KEY }}") | gpg --batch --import
+          gpg --list-secret-keys --keyid-format LONG
+      - name: Build standing data
         working-directory: output-data
         run: npm run build
-      - name: Release NPM Package
+      - name: Release NPM and Maven Packages
```

Comment:
I kind of liked them being in a separate step(if one package fails to release the other stays green), did it break Maven release?

Reply:
There are two main reasons that I wanted to bring them into one step:
* Using the `semantic-release` tool to do the maven publishing means that it'll only trigger the maven publish if the version has been changed (whereas before we were getting failures in the pipeline because the maven step was getting triggered and then trying to publish a version number that already exists in the repository)
* To prevent the rest of the workflow (i.e. the automatic PR generation) from executing if _either_ of them fails - I don't think it's an ideal situation to have node and java using different versions of the standing data, I think we want to try and keep everything consistent at that stage.

Does that make sense? Happy to hear counter-arguments! 😅

Reply:
Yep, I'm happy with this!